### DISCUSSION TITLE
通过vc-primitive-tileset加载的模型如何获取properties

### DISCUSSION THREAD

**Discussion by 987763485** [zouyaoji/vue-cesium#540] [Q&A]
![WX20240329-202931@2x](https://github.com/zouyaoji/vue-cesium/assets/22728641/8ccb41e5-20a4-46a1-955f-f52e901e5480)

我通过vc-selection-indicator打印的e.cesiumObject.properties和e.properties都为null，我怎样才能获取模型中的id和name属性

**Comment by zouyaoji** [comment]
```
const feature= e.pickedFeature as Cesium.Cesium3DTileFeature
console.log(feature.getProperty('你的属性字段名'))
```

**Reply by 987763485** [reply]
感谢
